Include/arm_nnfunctions.h

```diff
@@ -1754,6 +1764,78 @@ arm_cmsis_nn_status arm_svdf_state_s16_s8(const cmsis_nn_context *input_ctx,
                                           const cmsis_nn_dims *output_dims,
                                           int8_t *output_data);
 
+/**
+ * @defgroup LSTM LSTM Layer Functions
+ *
+ */
+
+/**
+ * @brief LSTM unidirectional function with 8 bit input and output and 16 bit gate output
+ * Peephole connections, projection, clipping, combined input/forget gate and layer normalization are not supported.
```

Comment:
For the generate pack comments below, you can run dxy.in script to reproduce it locally..It would verify the arguments against the comments in other places as well..